steembar.py

```diff
@@ -36,11 +38,29 @@ def steemit_api(steemit_name):
     url = '{}get_state?path=@{}'.format(API, steemit_name)
     return fetch(url)
 
-def get_vp(steemit_name):
+
+def get_vp_rp(steemit_name):
```

Comment:
Here is my function added to get the voting power and the reputation of the user